build.gradle

```diff
@@ -48,7 +48,7 @@ ext {
     nettyTcnativeBoringsslVersion = '2.0.48.Final'
     snappyVersion = '1.1.8.4'
     zstdVersion = '1.5.0-4'
-    mongoCryptVersion = '1.5.2'
+    mongoCryptVersion = '1.6.0-SNAPSHOT'
```

Comment:
Consider pinning to the `1.6.0-alpha0` release.

Reply:
Yes, will do that before merging.  I haven't release mongodb-crypt binding yet for the alpha

Reply:
Just released: https://repo1.maven.org/maven2/org/mongodb/mongodb-crypt/1.6.0-alpha0/

Reply:
Updated dependency